eos/form-factors/parametric-bgjvd2019-impl.hh

```diff
@@ -0,0 +1,1779 @@
+/* vim: set sw=4 sts=4 tw=120 et foldmethod=syntax : */
+
+/*
+ * Copyright (c) 2018, 2019 Danny van Dyk
+ *
+ * This file is part of the EOS project. EOS is free software;
+ * you can redistribute it and/or modify it under the terms of the GNU General
+ * Public License version 2, as published by the Free Software Foundation.
+ *
+ * EOS is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#ifndef MASTER_GUARD_EOS_FORM_FACTORS_PARAMETRIC_BGJvD2019_IMPL_HH
+#define MASTER_GUARD_EOS_FORM_FACTORS_PARAMETRIC_BGJvD2019_IMPL_HH 1
```

Comment:
Can you use `EOS_GUARD... for consistency ?

Reply:
Done.